localstack-core/localstack/services/kms/models.py

```diff
@@ -314,6 +329,9 @@ def decrypt(
             self.crypto_key.key_material, ciphertext.ciphertext, ciphertext.iv, ciphertext.tag, aad
         )
 
+    def decrypt_rsa(self, encrypted: bytes) -> bytes:
+        return _decrypt_rsa(self.crypto_key.private_key, encrypted)
+
```

Comment:
Any specific reason we keep the other helper function `_decrypt_rsa` within here now? 🤔 Considering there is no additional logic or abstraction we perform additionally before calling it we can move the entire thing here itself.

Reply:
@sannya-singal good call - no I just got a bit of tunnel vision 👍